README.md

```diff
@@ -76,18 +88,43 @@ Each zkVM benchmark implementation follows a common pattern:
     ./scripts/download-and-extract-fixtures.sh
     ```
 
-3. **(Optionally)Patching Precompiles**: Each zkVM, for efficiency purposes requires particular dependencies to be patched.
+3. **Generate Benchmark Input Files:**
+
+    Navigate to `crates/witness-generator/` and generate fixture files:
+
+    ```bash
+    cd crates/witness-generator
+    cargo run --release -- tests --include Prague --include cold
+    
+    # Or generate from RPC
+    cargo run --release -- rpc --last-n-blocks 2 --rpc-url <your-rpc-url>
+    ```
+
+    This creates individual `.json` files in the `zkevm-fixtures-input/` directory that will be consumed by the benchmark runner.
+
+4. **(Optionally) Patching Precompiles**: Each zkVM, for efficiency purposes requires particular dependencies to be patched.
 This repository contains an `xtask` that will automate this process by calling `cargo <zkvm-name>`. See `.config/cargo.toml` for how this is setup and `precompile-patches` for the patches that each zkVM requires.
 
 > Note: This is optional as it should be automatically done when the benchmarks are ran.
 
-4. **Run benchmark**: Navigate to `crates/ere-hosts/` and follow the readme instructions to run benchmarks for specific zkVMs.
+5. **Run Benchmarks:**
+
+    Navigate to `crates/ere-hosts/` and run benchmarks using the generated fixture files:
+
+    ```bash
+    cd crates/ere-hosts
+    cargo run --release --features sp1
+    cargo run --release --features "sp1,risc0"
+    ```
+
+    See the respective README files in each crate for detailed usage instructions.
 
 ## Supported zkVM Benchmarks
 
 | zkVM             | Guest Path            | Host Path           | Metrics Output             |
 | ---------------- | --------------------- | ------------------- | -------------------------- |
 | **Succinct SP1** | `ere-guests/sp1/`     | `crates/ere-hosts/` | `zkevm-metrics/sp1/`       |
-| **zkMIPS**       | `ere-guests/zkm/`     | `crates/zkevm-zkm/` | `zkevm-metrics/zkm/`       |
```

Comment:
Removing since [not supported](https://github.com/eth-act/zkevm-benchmark-workload/issues/8#issuecomment-2994198722).